packages/victory-area/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # victory-area
 
+## 36.6.0
+
+### Patch Changes
+
+- Updated dependencies []:
```

Comment:
So, something is up with the changelog plugin. We're supposed to have `[#COMMIT_HASH]` here, not `[]`.

Reply:
Looks to be fixed 👍